services/app-api/form/submitAny.js

```diff
@@ -68,42 +63,15 @@ export const submitAny = async (event, config) => {
       }
     }
 
-    // id validations, can be many for different parts of IDs
-    const promises = config.idExistValidations.map(
-      async ({ idMustExist, errorLevel, existenceRegex, validateParentId }) => {
-        let checkingNumber = data.componentId;
-        if (validateParentId && typeof config.getParentInfo == "function") {
-          [checkingNumber] = config.getParentInfo(data.componentId);
-        }
-
-        if (existenceRegex !== undefined) {
-          checkingNumber = data.componentId.match(existenceRegex)[0];
-        }
-        console.log("ID to check for existence: ", checkingNumber);
-
-        const isThere = await packageExists(checkingNumber);
-        if (idMustExist && !isThere) {
-          return [errorLevel, RESPONSE_CODE.ID_NOT_FOUND];
-        }
-        if (!idMustExist && isThere) {
-          return [errorLevel, RESPONSE_CODE.DUPLICATE_ID];
-        }
-      }
-    );
-
-    const results = await Promise.all(promises);
-    console.log("ID Matching results are: ", results);
-    results
-      .filter((errorMsg) => errorMsg != undefined)
-      .forEach((errorMsg) => {
-        console.log("theLevel: ", errorMsg[0]);
-        console.log("theCode: ", errorMsg[1]);
-        if (errorMsg[0] === "error") {
-          throw errorMsg[1];
-        } else {
-          warningsInCMSNotice.push(errorMsg[1]);
-        }
-      });
```

Comment:
I love this cleanup! At first i was thrown by warningsInCmsNotice being dropped......but since we no longer have warnings in package view it makes sense.